### DISCUSSION TITLE
MIDI Object for Convolution reverb

### DISCUSSION THREAD

**Discussion by fabiomancini67** [GrandOrgue/grandorgue#2005] [Developers]
It would be nice to add a MIDI object for enabling the convolution reverb, so that it can be enabled or disabled by the organist as needed either from any MIDI command on the console or from any on-screen button. To complete this wonderful option, it would be nice if all the other functions of the reverb had their own MIDI objects to be controlled without going into the dedicated sub-window in the settings menu.

Translated with DeepL.com (free version)

**Comment by newVideophile** [comment]
Oh yes, oh yes, dear developers, please!

**Comment by fabiomancini67** [comment]
Thinking back to the request made, probably the easiest thing to do would be to make the reverb amount parameter (Gain) a MIDI object: putting it to 0 which would exclude the effect, effectively making it useless to disable it. I suppose that this is easiest because to enable and disable it in addition to the check box you need to click on the OK button.

**Comment by newVideophile** [comment]
I'm not a developer of GrandOrgue, but I'm not sure if this would help. In the GUI, you need to click on the OK button to make the gain change effective in any case.
On a side note, I have asked for this quite a long time ago, but this does not seem to be a priority of the developers. To the developers : don't get me wrong here, you are doing a great job, this is just an observation, and by no way a criticism. I know you're doing it for free and I'm very grateful to be able to use this amazing piece of software!

**Comment by fabiomancini67** [comment]
newVideophile you are right! 
Probably the best thing to do as first step is to have the possibility to store the informations about the use of the convolution reverb for  each organ differently, as already requested somewhere else. 
I'm conviced that the convolution reverb works very well (apart when you load a different sample set - you have to disable and enable it again or reload the sample set in order to get rid of an initial latency) and it can make a huge difference in using the software on little computer with 8 GB ram for example. This is how Sweelinq works, and I think that is a very good idea in order to have faster loading time, adjustable level of reverberation that can make sample sets clear or very wet, and lot of other possibilities. Also very old sample set with just one release sounds much better. 
And I agree with you about the wonderful works that all the developers are doing, as it is is already a dream and we can experiment with lot of different solutions, it's the only software that permit this!

**Reply by kerkovits** [reply]
I completely agree, as I am the author of that issue. https://github.com/GrandOrgue/grandorgue/issues/1368 I have also experienced the initial latency, but found that instead of reloading, a simple press on the MIDI panic button is sufficient to reduce the delay. That is a bit faster than a full reload.
